tests/cypress/tests/e2e/header.spec.js

```diff
@@ -9,10 +9,13 @@ const menuOptionLogOut = '[data-testid="header-menu-option-log-out"]';
 const cognitoLoginButton = "[data-testid='cognito-login-button']";
 
 beforeEach(() => {
-  cy.visit("/");
   cy.authenticate("stateUser");
 });
 
+afterEach(() => {
+  cy.visit("/");
```

Comment:
we could speed the runtime slightly by putting a conditional visit here.
`if (cy.location("pathname") !== "/") cy.visit("/")`

Reply:
adding this - thanks!